components/notes/notes.js

```diff
@@ -0,0 +1,27 @@
+import Link from "next/link";
+import BackButton from "../navigation/backbutton/backbutton";
+import { useRef, useEffect } from "react";
+
+export default function NoteFormular({}) {
+  const inputReference = useRef(null);
+  useEffect(() => {
+    inputReference.current.focus();
+  }, []);
+
+  return (
+    <form action="/send-data-here" method="post">
```

Comment:
You should process the form via JavaScript and prevent the default behavior. So please remove `action` and `method` and add an `onSubmit` event instead where you prevent the default behavior.